### PR TITLE
Possible use of context manager using 'with' for file operations

### PR DESCRIPTION
**Pull request by kenjbaugh**
By implementing the context manager on line 29 using the 'with' statement, we can write to the file using the 'f' variable, which will automatically handle closing out the file once opening.

### CODE REVIEW

File: enJsonCounter.py [hunk 1]
```diff
@@ -13,195 +14,185 @@ def main():
     import re
     import collections
     import datetime
+    from contextlib import contextmanager
 
     #class imports
-    from classes.steps import Step3
-    from classes.steps import Step1and2
+    #from classes.steps import Step3
+    #from classes.steps import Step1and2
```
Comment (colelins) [inline enJsonCounter.py:21]: Should these be commented?

File: enJsonCounter.py [hunk 2]
```diff
@@ -13,195 +14,185 @@ def main():
...
+                    # Swap if the element found is greater
+                    # than the next element
+                    if arr[j].wordCount < arr[j+1].wordCount:
+                        arr[j], arr[j+1] = arr[j+1], arr[j]
 
-        wordsList = []
-        total = 0
+                    return arr
+
+            def step1and2func():
```
Comment (colelins) [inline enJsonCounter.py:65]: I think some of the indentation got off here

File: enJsonCounter.py [hunk 3]
```diff
@@ -13,195 +14,185 @@ def main():
...
 
-        start_time = time.time()
+                print("running report")
+                wordsList = []
+                total = 0
+                JsonData = json.load(open('exampleJSON.json', encoding='utf-8'))
```
Comment (colelins) [inline enJsonCounter.py:70]: JsonData instantiation never got uncommented and so this doesn't exist and is giving me squiglies

**Review by colelins — COMMENTED** [review]
I like the idea of using with! Because we were tweaking so much with this file there were a bunch of merge conflicts though. Before creating the PR you can run something like ``` git merge master``` to find/work out any conflicts. Also make sure you are able to run the program successfully! Lets try and run through this process together now we know some changes we can make. I'm interested in learning more about forks. Thanks for contributing!

File: enJsonCounter.py (hunk 1 above)
Comment (kenjbaugh) [reply enJsonCounter.py:21]: Absolutely, sounds like a plan. Thank you for the feedback.

**Comment by colelins** [comment]
Resolved with new PR.

**Pull request closed without merging**